File: go_fasthttp/app/src/rabbitmq/queues.go
package rabbitmq

const (
	ExchangeName      = "go-fasthttp"
	QueueGolang       = "go-fasthttp"
	QueueSystemGolang = "system-go-fasthttp"
)

// GetQueueName возвращает название очереди в зависимости от типа события
func GetQueueName(isSystem bool) string {
	if isSystem {
		return QueueSystemGolang
	}
	return QueueGolang
}

File: rust_hyper/app/src/rabbitmq/queues.rs
pub const EXCHANGE_NAME: &str = "rust-hyper";
pub const QUEUE_HYPER: &str = "rust-hyper";
pub const QUEUE_SYSTEM_HYPER: &str = "system-rust-hyper";

pub fn get_queue_name(is_system: bool) -> &'static str {
    if is_system {
        QUEUE_SYSTEM_HYPER
    } else {
        QUEUE_HYPER
    }
}
